File: python/longest_continuous_increasing_subsequence.py
from typing import List


class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        current, result = 1, 1
        for index in range(1, len(nums)):
            if nums[index] > nums[index - 1]:
                current += 1
                result = max(result, current)
            else:
                current = 1
        return result

File: src/LongestContinuousIncreasingSubsequence.java
public class LongestContinuousIncreasingSubsequence {
    public int findLengthOfLCIS(int[] nums) {
        int result = 1, current = 1;
        for (int index = 1 ; index < nums.length ; index++) {
            if (nums[index] > nums[index - 1]) {
                current++;
                result = Math.max(result, current);
            } else {
                current = 1;
            }
        }
        return result;
    }
}
